pages/conference-day.tsx

```diff
@@ -0,0 +1,233 @@
+import React from 'react'
+import { GetServerSideProps, NextPage } from 'next'
+import dynamic from 'next/dynamic'
+import Link from 'next/link'
+import { format } from 'date-fns'
+import { useConfig } from 'Context/Config'
+import { Session } from 'config/types'
+import { Main } from 'layouts/main'
+import { getCommonServerSideProps } from 'components/utils/getCommonServerSideProps'
+import { fetchSessions } from 'components/utils/useSessions'
+import { useSessionGroups } from 'components/utils/useSessionGroups'
+import { StyledList, Text } from 'components/global/text'
+import { SafeLink } from 'components/global/safeLink'
+import { roomLocations } from 'components/venueMapData'
+import Image from 'next/image'
+
+const VenueMap = dynamic(() => import('components/venueMap'), { ssr: false })
+
+interface ConferencePageProps {
+  sessions?: Session[]
+  sessionId?: string
+}
+
+const ConferenceDayPage: NextPage<ConferencePageProps> = ({ sessions }) => {
+  const { conference, dates } = useConfig()
+  if (dates.IsInProgress) {
+    // dummy
+  }
+
+  const { currentSessionGroup } = useSessionGroups(sessions)
+
+  if (currentSessionGroup && currentSessionGroup.sessions.length > 0) {
+    // NB: This is quite brittle, as it assumes that the list of current sessions is returned in exactly the same order as the definition of rooms
+    currentSessionGroup.sessions.map(function (session: Session, i) {
+      const presenters = []
+      session.Presenters.map((p) => {
+        presenters.push(p.Name)
+      })
+      if (!roomLocations.features[i]) return
+      roomLocations.features[i].properties.currentEvent = {
+        eventId: session.Id,
+        eventName: session.Title,
+        eventPresenters: presenters.join(', '),
+      }
+    })
+  }
+
+  return (
+    <Main title="Conference Day" description="Conference day information">
+      <h1>
+        {conference.Name} {format(conference.Date, 'y')} Conference Day Information
+      </h1>
+      {conference.Handbook && (
+        <p>
+          You can also download much of the information on this page as a PDF:&nbsp;
+          <a className="btn btn-pdf" href={'/static/docs/' + conference.Handbook}>
+            Download handbook (PDF)
+          </a>
+        </p>
+      )}
+      <h2>Getting There</h2>
```

Comment:
Are these FAQs covered in the FAQ page? Should it be a shared component or something?

Reply:
This used to have duplicated content from the Venue page, so I've now just linked to that.